From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Tue, 18 Oct 2016 12:26:10 +0200
Subject: [PATCH] Revert "Point to specific snapshot of mos repositories"

Armband does not publish snapshots yet.

This reverts commit fe3fc91c23ace520f32830b6af5205c39088f922.
---
 build/Makefile                       | 4 +---
 build/f_isoroot/f_repobuild/Makefile | 4 ++--
 2 files changed, 3 insertions(+), 5 deletions(-)

diff --git a/build/Makefile b/build/Makefile
index 5069def..59fa918 100644
--- a/build/Makefile
+++ b/build/Makefile
@@ -63,9 +63,7 @@ export MIRROR_UBUNTU_ROOT := $(shell echo -n '/' ; echo "$(MIRROR_UBUNTU_URL)" |
 export LATEST_MIRROR_ID_URL := http://$(shell ./select_closest_fuel_mirror.py)
 
 export MIRROR_MOS_UBUNTU ?= $(shell echo "$(LATEST_MIRROR_ID_URL)" | cut -d'/' -f3)
-#export LATEST_TARGET_UBUNTU := $(shell curl -sSf "$(MIRROR_MOS_UBUNTU)/mos-repos/ubuntu/$(MOSVERSION).target.txt" | head -1)
-# Fuel 9.0
-export LATEST_TARGET_UBUNTU := snapshots/9.0-2016-06-23-164100-copy
+export LATEST_TARGET_UBUNTU := $(shell curl -sSf "$(MIRROR_MOS_UBUNTU)/mos-repos/ubuntu/$(MOSVERSION).target.txt" | head -1)
 export MIRROR_MOS_UBUNTU_ROOT := /mos-repos/ubuntu/$(LATEST_TARGET_UBUNTU)
 
 export LATEST_TARGET_CENTOS := $(shell curl -sSf "$(LATEST_MIRROR_ID_URL)/mos-repos/centos/mos$(MOSVERSION)-centos7/os.target.txt" | head -1)
diff --git a/build/f_isoroot/f_repobuild/Makefile b/build/f_isoroot/f_repobuild/Makefile
index 8beb882..ad1a934 100644
--- a/build/f_isoroot/f_repobuild/Makefile
+++ b/build/f_isoroot/f_repobuild/Makefile
@@ -53,9 +53,9 @@ release:nailgun
 	@cp -Rp nailgun ../release/opnfv/nailgun
 	@cp fuel_bootstrap_cli.yaml ../release/opnfv/
 
-############################################################################
+#############################################################################
 # Cache operations - only used when building through ci/build.sh
-############################################################################
+#############################################################################
 
 # Create a unique hash to be used for getting and putting cache, based on:
 #   - Year and week (causing the cache to be rebuilt weekly)
